hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -4289,8 +4249,9 @@ public void checkLeaderStatus() throws OMNotLeaderException,
   /**
    * Return if Ratis is enabled or not.
    */
+  // FIXME remove
```

Comment:
TODO

Reply:
This is for HDDS-12161, added issue number in the comment.